PriceCalculatorKata/PriceCalculator.cs

```diff
@@ -5,73 +5,29 @@ namespace PriceCalculatorKata;
 
 public static class PriceCalculator
 {
-    private const int DefaultTax = 20;
-    private const int DecimalPrecision = 2;
-
-    private static int _taxPercentage = DefaultTax;
-    public static int TaxPercentage
-    {
-        get => _taxPercentage;
-        set
-        {
-            if (!value.IsValid())
-                throw new ArgumentException("Invalid Tax Percentage", $"{value}");
-            _taxPercentage = value;
-        }
-    }
     
-    private static int _discountPercentage;
-    public static int DiscountPercentage
-    {
-        get => _discountPercentage;
-        set
-        {
-            if (!value.IsValid())
-                throw new ArgumentException("Invalid Discount Amount!", $"{value}");
-            _discountPercentage = value;
-        }
-    }
-
     public static void DisplayPrice(Product product)
     {
-        var totalPrice = CalculateTotalPrice(product.Price);
-        var discountToPrint = DiscountPercentage > 0 ? $"%{DiscountPercentage}" : "no";
+        var totalPrice = CalculateTotalPrice(product);
+        var discountToPrint = DiscountCalculator.DiscountPercentage > 0 ? $"%{DiscountCalculator.DiscountPercentage}" 
```

Comment:
is this class responsible for this logic (deciding whether there is a discount or not based on the DiscountPercentage property value? or should it be encapsulated?

Reply:
> is this class responsible for this logic (deciding whether there is a discount or not based on the DiscountPercentage property value? or should it be encapsulated?

No this should be handled in separate methods where they handle returning the correct value. I've just pushed a fix for this.